packages/app/contexts/ModalContext.tsx

```diff
@@ -18,6 +18,7 @@ export enum ModalId {
   SUCCESS_STACK_TOAST = "success_stack_toast",
   STACK_APPROVE_PROCESSING = "stack_approve_processing",
   STACK_CREATION_PROCESSING = "stack_creation_processing",
+  BETA_NFT = "beta_nft",
```

Comment:
beta nft what? 

I think we should be a bit more descriptive here. What's the point of the modal?

some examples that I think are better:
`beta_nft_blocker ` || `beta_nft_required` || `how_to_mint_beta_nft_blocker`